package.js

```diff
@@ -28,8 +27,7 @@ const DEFAULT_OPTS = {
     '^/test($|/)',
     '^/release($|/)',
     '^/main.development.js'
-  ].concat(devDeps.map(name => `/node_modules/${name}($|/)`))
```

Comment:
This is done by default as of 8.0.0 (and in a better way, because it also handles dependencies of devDependencies).